### PR TITLE
Re-enable skipped tests

### PR DESCRIPTION
**Pull request by NehaBroad**
Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: e2e/tests/datasets/dataset-export-py-notebook.spec.ts [hunk 1]
```diff
@@ -45,6 +41,6 @@ describe('Export Dataset to Notebook Test', () => {
 
     // Delete notebook.
     const analysisPage = await notebookPreviewPage.goAnalysisPage();
-    await analysisPage.deleteResource(notebookName, ResourceCard.Notebook);
+    await analysisPage.deleteResourceFromTable(notebookName, ResourceCard.Notebook);
```
Comment (jmthibault79) [inline e2e/tests/datasets/dataset-export-py-notebook.spec.ts:44]: Should we replace all existing instances of `deleteResource()` and remove it?

(for a future PR)

File: e2e/tests/notebook/notebook-create-r.spec.ts [hunk 2]
```diff
@@ -58,22 +56,19 @@ describe('Create R kernel notebook', () => {
...
     await openTab(page, Tabs.Analysis, analysisPage);
 
     // Start clone notebook.
     const cloneNotebookName = `Duplicate of ${rNotebookName}`;
-    await analysisPage.duplicateNotebook(rNotebookName);
+    await analysisPage.duplicateNotebookViaTable(rNotebookName);
 
     // Rename notebook clone.
     const newNotebookName = makeRandomName('r-cloneNotebook');
-    const modalTextContents = await analysisPage.renameResource(
```
Comment (jmthibault79) [inline e2e/tests/notebook/notebook-create-r.spec.ts:76]: renameResource() is no longer used, so we should remove it.

File: e2e/tests/notebook/notebook-create-r.spec.ts [hunk 3]
```diff
@@ -58,22 +56,19 @@ describe('Create R kernel notebook', () => {
...
-  test.skip('Duplicate rename delete notebook', async () => {
+  test('Duplicate rename delete notebook', async () => {
     await loadWorkspace(page, workspaceName);
 
     const analysisPage = new WorkspaceAnalysisPage(page);
     await openTab(page, Tabs.Analysis, analysisPage);
 
     // Start clone notebook.
     const cloneNotebookName = `Duplicate of ${rNotebookName}`;
-    await analysisPage.duplicateNotebook(rNotebookName);
```
Comment (jmthibault79) [inline e2e/tests/notebook/notebook-create-r.spec.ts:72]: duplicateNotebook() is no longer used, so we should remove it

File: e2e/tests/notebook/notebook-create-r.spec.ts [hunk 4]
```diff
@@ -82,21 +77,22 @@ describe('Create R kernel notebook', () => {
 
     // Notebook card with new name is found.
     const dataResourceCard = new DataResourceCard(page);
-    let cardExists = await dataResourceCard.cardExists(newNotebookName, ResourceCard.Notebook);
+    let cardExists = await dataResourceCard.resourceExistsInTable(newNotebookName);
+    //let cardExists = await dataResourceCard.cardExists(newNotebookName, ResourceCard.Notebook);
     expect(cardExists).toBe(true);
 
     // Notebook card with old name is not found.
-    cardExists = await dataResourceCard.cardExists(cloneNotebookName, ResourceCard.Notebook);
```
Comment (jmthibault79) [inline e2e/tests/notebook/notebook-create-r.spec.ts:89]: please remove cardExists()

**Review by jmthibault79 — APPROVED** [review]
This is great!  Thank you for restoring this test coverage.  Before merging, please do some additional cleanup - remove these functions:
* duplicateNotebook()
* renameResource()
* cardExists()

In a future PR, please also investigate whether `deleteResource()` can also be replaced and removed.

File: e2e/tests/datasets/dataset-export-py-notebook.spec.ts (hunk 1 above)
Comment (NehaBroad) [reply e2e/tests/datasets/dataset-export-py-notebook.spec.ts:44]: there still are some skipped tests, will remove it in upcoming PR

File: e2e/tests/notebook/notebook-create-r.spec.ts (hunk 3 above)
Comment (NehaBroad) [reply e2e/tests/notebook/notebook-create-r.spec.ts:72]: Done

File: e2e/tests/notebook/notebook-create-r.spec.ts (hunk 2 above)
Comment (NehaBroad) [reply e2e/tests/notebook/notebook-create-r.spec.ts:76]: done

File: e2e/tests/notebook/notebook-create-r.spec.ts (hunk 4 above)
Comment (NehaBroad) [reply e2e/tests/notebook/notebook-create-r.spec.ts:89]: done replaced with notebookExist

**Pull request merged**